includes/shortcode-basepage.php

```diff
@@ -92,7 +92,7 @@ public function anspress_sc( $atts, $content = '' ) {
 		echo '</div>';
 		// Linkback to author.
 		if ( ! ap_opt( 'author_credits' ) ) {
-			echo '<div class="ap-cradit">' . esc_attr__( 'Question and answer is powered by', 'anspress-question-answer' ) . ' <a href="https://anspress.io" target="_blank">AnsPress.io</a></div>';
+			echo '<div class="ap-cradit">' . esc_attr__( 'Question and answer is powered by', 'anspress-question-answer' ) . ' <a href="https://anspress.io" traget="_blank">AnsPress.io</a></div>';
```

Comment:
This seems to be a typo.

Reply:
Yep - and it will break UI, so needs fixing.